network-config-analyzer/SchemeRunner.py

```diff
@@ -82,14 +83,22 @@ def _add_config(self, config_entry, peer_container_global):
 
         ns_list = config_entry.get('namespaceList')
         pod_list = config_entry.get('podList')
-        if ns_list or pod_list:  # a local resource file exist
+        srv_list = config_entry.get('serviceList')
```

Comment:
We want to avoid adding this new entry of `serviceList`, and instead have the services within the current `podList`